orderapi/src/main/java/fi/hel/verkkokauppa/order/service/accounting/AccountingSlipService.java

```diff
@@ -250,9 +352,39 @@ public Map<String, List<OrderItemAccountingDto>> getSummedOrderItemAccountingsFo
         return summedAccountingsForCompanyCode;
     }
 
-    private List<AccountingSlipRowDto> createAccountingSlipRowDtos(List<OrderItemAccountingDto> summedItemAccountings, String accountingSlipId,
-                                                                   String lineText, int rowCountOffset) {
+    public Map<String, List<RefundItemAccountingDto>> getSummedRefundItemAccountingsForDate(List<String> accountingsForDate) {
+        List<RefundItemAccountingDto> accountingItemsForDate = new ArrayList<>();
+        if (accountingsForDate == null) {
+            return new HashMap<>();
+        }
+
+        for (String id : accountingsForDate) {
```

Comment:
id rename this to refundItemId or what is this id?

Reply:
refundAccountingId, renamed